scipy/signal/filter_design.py

```diff
@@ -3504,6 +3504,196 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, ab=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    Parameters
+    ----------
+    w0 : float
+        A scalar that must satisfy  0 < w0 < 1. For notch filters,
```

Comment:
Better to start the description from the definition, like "Normalized frequency to remove from the filtered signal". And then explain its properties.